libminifi/include/core/FlowFile.h

```diff
@@ -134,7 +135,9 @@ class FlowFile : public CoreComponent, public ReferenceContainer {
    * @param value value to set
    * @return result of finding key
    */
-  bool getAttribute(std::string key, std::string& value) const;
+  bool getAttribute(const std::string& key, std::string& value) const;
+
+  utils::optional<std::reference_wrapper<const std::string>> getAttribute(const std::string& key) const;
```

Comment:
Good observation, this was cherry-picked from #911 hoping that it would be merged much much earlier than this PR, whereas it is still open. I think I will just close that PR and move the change here.

Reply:
Updated.